content/blog/2020/05/2020-05-12-uiux-hackfest-announcement.adoc

```diff
@@ -0,0 +1,126 @@
+  image: /images/post-images/jenkins-is-the-way/jenkins-is-the-way-hackfest-opengraph.png
+---
+
+image:/images/post-images/jenkins-is-the-way/jenkins-is-the-way.png[Jenkins Is The Way, role=center, float=right, height=284]
+
+On behalf of the Jenkins _User Experience_, _Documentation_ and _Advocacy and Outreach_ special interest groups,
+we are happy to announce the link:/events/online-hackfest/2020-uiux/[online UI/UX hackfest] on May 25-29!
+Everyone is welcome to participate, regardless of their Jenkins development experience.
+
+The goal to get together and to work on improving Jenkins user experience,
+including but not limited to user interface and user documentation.
+We also invite you to share experiences about Jenkins and the recent UI/UX improvements.
+The event follows the link:/blog/2020/04/30/jenkins-is-the-way/[Jenkins is the Way] theme,
+most active contributors will get special edition swag and prizes!
+
+image:/images/post-images/jenkins-is-the-way/register-button.png[link="https://forms.gle/MrkASJagxNvdXBbdA", role=center, align=text-center, height=32]
+
+== Event plan
+
+This hackfest is NOT a _hackathon_.
+We do not expect participants to dedicate all their time during the event timeframe.
+Instead of that, we organize it as a hop-in/hop-off event.
+Everybody can spend as much time as they are willing to dedicate.
+Spending a few days or just a few hours is fine, any contributions matter regardless of their size.
+Jenkins development experience is not required,
+we have newcomer-friendly stories for those who want to start contributing to the project.
+we will also have a 24/7 link:https://gitter.im/jenkinsci/hackfest[jenkinsci/hackfest] Gitter chat for Q&A and coordination between contributors.
+
+There will be link:/events/online-hackfest/2020-uiux/#tracks-and-project-ideas[3 main tracks]:
+
+* **User Interface** -
+  Improve look&feel and accessibility for Jenkins users;
+  work on new read-only interface for instances managed with configuration as code;
+  create and update Jenkins themes;
+  and many other topics.
+  This track is coordinated by the link:/sigs/ux/[UX SIG].
+* **User Documentation** -
+  Improve and create new link:/doc/book/[user documentation], link:/doc/tutorials/[tutorials] and link:/solutions/[solution pages].
+  Also, there is ongoing documentation migration from Wiki to jenkins.io and plugin repositories.
+  This track is coordinated by the link:/sigs/docs/[Documentation SIG].
+* **Spread the word** -
+  Write user stories for link:https://jenkinsistheway.io/[Jenkins Is The Way] site and the link:/blog[Jenkins blog],
+  post about your Jenkins user experience and new features,
+  record overview and HOWTO videos, etc.
+  This track is coordinated by the link:/sigs/advocacy-and-outreach/[Advocacy and Outreach SIG].
+
+We are working on publishing project ideas and issues for the listed tracks.
+The current list can be found link:/events/online-hackfest/2020-uiux/#tracks-and-project-ideas[here],
```

Comment:
It's more accessible to see what a link is rather than 'here'

```suggestion
The current list can be found on the link:/events/online-hackfest/2020-uiux/#tracks-and-project-ideas[UI / UX hackfest event page],
```